src/components/sbb-tooltip/sbb-tooltip.tsx

```diff
@@ -352,10 +352,12 @@ export class SbbTooltip implements ComponentInterface {
   // Set focus on the first focusable element.
   private _setTooltipFocus(): void {
     this._prevFocusedElement = document.activeElement as HTMLElement;
-    this._firstFocusable = this._element.querySelector(IS_FOCUSABLE_QUERY);
+    this._firstFocusable =
```

Comment:
Why was this change necessary?

Reply:
@dauriamarco : we did this because the first focusable element might be present in the content provided by the consumer, otherwise it's the close button (inside the shadowRoot)

Reply:
Thank you for quoting me 😄. I also have an additional consideration: I'm not sure whether just using `this._element.querySelector(IS_FOCUSABLE_QUERY)` is enough or if we should use the `getFocusableElements` function from the `focus.ts` file. What do you think?

Reply:
I think `IS_FOCUSABLE_QUERY` is enough in this use case, as we are primarily interested in the light DOM elements. Maybe it might make sense to directly query for the close button?